src/widgets/main/ui/ExpoListContainer/index.tsx

```
@@ -1,53 +1,64 @@
-import { useMemo, useState } from 'react';
+import { useEffect, useState } from 'react';
 import { ExpoListItem } from '@/entities/main';
 import withLoading from '@/shared/hocs/withLoading';
-import { sortedData } from '@/shared/model/sortedData';
-import { useExpoList } from '@/shared/queries/useExpoList';
-import { ExpoItem } from '@/shared/types/main/type';
-import SortFilter from '@/shared/ui/SortFilter';
 import { filterOptions } from '../../constant/filterOptions';
 import EmptyExpoList from '../EmptyExpoList';
+import FormFilter from '../FormFilter';
+import { useExpoList } from '@/shared/queries/useExpoList';
```

ESLint

[error] `@/shared/queries/useExpoList` import should occur before import of `../../constant/filterOptions`

(import/order)